pkg/process/checks/process.go

```diff
@@ -137,7 +135,24 @@ func (p *ProcessCheck) Init(syscfg *SysProbeConfig, info *HostInfo, oneShot bool
 
 	p.notInitializedLogLimit = log.NewLogLimit(1, time.Minute*10)
 
-	networkID, err := cloudproviders.GetNetworkID(context.TODO())
+	var tu *net.RemoteSysProbeUtil
+	var err error
+	if syscfg.NetworkTracerModuleEnabled {
+		// Calling the remote tracer will cause it to initialize and check connectivity
+		tu, err = net.GetRemoteSystemProbeUtil(syscfg.SystemProbeAddress)
+		if err != nil {
+			log.Warnf("could not initiate connection with system probe: %s", err)
+		} else {
+			// Register process agent as a system probe's client
+			// This ensures we start recording data from now to the first call to `Run`
+			err = tu.Register(ProcessAgentClientID)
```

Comment:
will this cause memory issues by causing the system-probe to retain connections for the `ProcessAgentClientID` ?

Reply:
I removed this